pkg/controllers/node/termination/terminator/metrics.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package terminator
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	crmetrics "sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+var (
+	PodEvictedCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "karpenter",
+			Subsystem: "pods",
+			Name:      "evicted",
+			Help:      "Number of pods evicted by Karpenter during node termination",
+		},
+		[]string{},
```

Comment:
Should we consider setting NodePool as a label here? We do everywhere else and I think it generally makes sense to want to correlate the number of evictions that came from a given NodePool

Reply:
Consider also doing namespace here as a label.